src/components/scenes/PluginViewScene.js

```diff
@@ -349,6 +418,11 @@ class PluginView extends React.Component<PluginProps, PluginState> {
     }
   }
 
+  webviewLoaded = () => {
+    const edgeProvider = new EdgeProvider(this.props.plugin, this.props.currentState, this.props.thisDispatch)
+    bridgifyObject(edgeProvider)
```

Comment:
Not needed if `class EdgeProvider extends Bridgeable`

Reply:
done